wordpress-mobile/WordPress-Android
# [Gutenberg] Highlight text bug fixes

**Related PRs:**
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/6517
- https://github.com/WordPress/gutenberg/pull/57650
- https://github.com/wordpress-mobile/AztecEditor-Android/pull/1073

Fixes several issues related to the Highlight text formatting in the Gutenberg editor.

-----

## To Test:

Please check the testing instructions in the [Gutenberg PR description](https://github.com/WordPress/gutenberg/pull/57650).

-----

## Regression Notes

1. Potential unintended areas of impact

    - It should only affect the editor.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing, and integration tests in both Aztec and Gutenberg.

3. What automated tests I added (or what prevented me from doing so)

    - Visual E2E tests added in Gutenberg Mobile.

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
